### PR TITLE
improve README

### PR DESCRIPTION
**Pull request by Ravna**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -2,8 +2,19 @@
...
+## Availability
+
+- You can find information about the project as a whole, including [papers](https://securedna.org/research) and a [live demo](https://securedna.org/demo/), at the [main site](https://securedna.org).
+- Precompiled `synthclient` binaries and certificate tools are available for [Ubuntu amd64 and arm64](https://github.com/SecureDNA/ppa).
+- Docker/podman container images are also available:
+  - As a minimal-size image containing just `synthclient`: `docker pull ghcr.io/securedna/synthclient`
+  - As an image containing both `synthclient` and the tools required to request a certificate and generate authorization tokens: `docker pull ghcr.io/securedna/synthclient-tools`
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
+  - You can compile this source and run test servers using test databases and your own test certificates.  To screen with a real database, see [below](#performing-screening).
+  - [quickdna](https://github.com/SecureDNA/SecureDNA) is a general-purpose DNA manipulation tool but does not have to be separately downloaded to build or run the servers or client.
```
Comment (lynn) [inline profile/README.md]:
```suggestion
  - [quickdna](https://github.com/SecureDNA/quickdna) is a general-purpose DNA manipulation tool but does not have to be separately downloaded to build or run the servers or client.
```

Comment (lynn) [reply profile/README.md]: Maybe quickdna is better described as a library than a tool. Also, if it isn't relevant to the main product maybe we should move this link to the bottom of the README, under a `## See also` or `## Other projects`.

File: profile/README.md [hunk 2]
```diff
@@ -2,8 +2,19 @@
...
+
+## Performing screening
+
+- To use the production servers and hence screen using real databases, you'll need to [obtain certificates](https://securedna.org/start/) by following the [quickstart instructions](https://pages.securedna.org/production/assets/Synthclient-quickstart.pdf).
+- For automated use, consult our [API documention](https://pages.securedna.org/production/assets/Synthclient-API.pdf)
+- Results can also be visualized at [your local `synthclient`](http://localhost:1234/); see the [quickstart](https://pages.securedna.org/production/assets/Synthclient-quickstart.pdf) for more information.
```
Comment (lynn) [inline profile/README.md]: (Do we really default to port 1234? I thought it was 80.) I'd be textually explicit about linking to localhost here:

```suggestion
- Results can also be visualized at <http://localhost:1234/>; see the [quickstart](https://pages.securedna.org/production/assets/Synthclient-quickstart.pdf) for more information.
```

Comment (Ravna) [reply profile/README.md]: I'm not sure where the 1234 came from.  If it's really 80 (can you check?), we should say so, and yes, I agree re being textually explicit.

Comment (lynn) [reply profile/README.md]: It's 80 :)

Comment (Ravna) [reply profile/README.md]: OK.  Modify your suggestion & I'll commit it. :)

**Pull request merged**